mscore/musescore.cpp

```diff
@@ -1,4 +1,4 @@
-//=============================================================================
+﻿//=============================================================================
```

Comment:
Why this changed? what is it about?

Reply:
I have no idea, I've tried to copy and paste the old line into my branch but it wont go away.

Reply:
Then just exclude it from the commit

Reply:
What the heck, just keep it. Along with the other formatting changes

Reply:
Viewing this change with git directly shows that it adds a [byte-order mark](https://en.wikipedia.org/wiki/Byte_order_mark) to the beginning of the file:
```
-//=============================================================================
+<U+FEFF>//=============================================================================
```

This is not something that would be wanted here, it is better not to keep this change. Byte-order mark can probably be disabled in your text editor (or IDE) settings, it is better to  disable it for files included in MuseScore project.

Reply:
Done, even though QtCreator was saying that it would not add a Byte-order mark to files that don't already have one.